index.bs

```diff
@@ -2269,14 +2269,18 @@ validated by the authenticator during the [=authenticatorGetAssertion=] operatio
 with the requested credential exactly matches the [=RP ID=] supplied by the client, and that the [=RP ID=] [=is a registrable
 domain suffix of or is equal to=] the [=effective domain=] of the RP's [=origin=]'s [=effective domain=].
 
-The `UP` flag SHALL be set if and only if the authenticator detected a user through an authenticator specific gesture. The
-`RFU` bits SHALL be set to zero.
+[=Authenticators=] <dfn for="authenticator data">perform the following steps to generate an [=authenticator data=] structure</dfn>:
 
-For attestation signatures, the authenticator MUST set the AT flag and include the <code>[=attestedCredentialData=]</code>. For
-authentication signatures, the AT flag MUST NOT be set and the <code>[=attestedCredentialData=]</code> MUST NOT be included.
+- Hash [=RP ID=] to generate the [=rpIdHash=].
```

Comment:
Should this also state that the hash function is SHA-256? It's specified in the linked definition of `rpIdHash`, so it may not be necessary.

Reply:
fixed.